src/condor_scripts/condor_upgrade_check

```diff
@@ -1,555 +1,151 @@
 #!/usr/bin/python3
-# HTCondor V9 to V10 check for incompatibilites script
-#   This is intended to check a current system before upgrade
-#   for known 'gotcha' breaking changes:
-#        1. IDToken TRUST_DOMAIN default value change
-#        2. Upgrade to PCRE2 breaking map file regular expressions
-#        3. The way to request GPU resources for a job
-#   This script can still be ran post upgrade as a post mortem
-#   Note: Requires root and HTCONDOR python bindings
+# Script to check for known gotchas that occur between
+# major versions of htcondor. This script will only check
+# for gotchas to the next version.
+
+# Imports Always needed
 import os
 import sys
-import json
-import platform
 import subprocess
-import shlex
-from shutil import which
-from tempfile import NamedTemporaryFile
-from pathlib import Path
+import textwrap
+import argparse
+from enum import Enum
+# Temporary needed imports (for a check)
+# End imports
 
 try:
     import htcondor
 except ImportError:
-    msg = """Failed to find HTCondor Python bindings.
-Please check your current Python environment or install the bindings if needed:
-https://htcondor.readthedocs.io/en/latest/apis/python-bindings/install.html"""
-    print(msg, file=sys.stderr)
+    print(
+        textwrap.dedent(
+            """
+            Failed to find HTCondor Python bindings.
+            Please check your current Python environment or install the bindings if needed:
+            https://htcondor.readthedocs.io/en/latest/apis/python-bindings/install.html
+            """
+        ),
+        file=sys.stderr,
+    )
     sys.exit(1)
 
-# Base special posix characters for PCRE2 regular expression character classes
-PCRE2_POSIX_CHARS = [
-    "[:alnum:]",
-    "[:alpha:]",
-    "[:ascii:]",
-    "[:blank:]",
-    "[:cntrl:]",
-    "[:digit:]",
-    "[:graph:]",
-    "[:lower:]",
-    "[:print:]",
-    "[:punct:]",
-    "[:space:]",
-    "[:upper:]",
-    "[:word:]",
-    "[:xdigit:]",
-]
 
+class Debug(Enum):
+    WARNING = 0
+    BASE = 1
+    DETAILED = 2
+    MAX = 4
```

Comment:
If I'm understanding this right, `-vv` and `-vvv` are equivalent (for now)?

Reply:
Yeah which would also be equivalent to `-vvvvvvvv`. I need to give some thoughts about how I want to maintain the tool output on increasing levels